pkg/daprrp/setup/operations.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2023 The Radius Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package setup
+
+import v1 "github.com/radius-project/radius/pkg/armrpc/api/v1"
+
+var operationList = []v1.Operation{
+	{
+		Name: "Applications.Dapr/operations/read",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Dapr",
+			Resource:    "operations",
+			Operation:   "Get operations",
+			Description: "Get the list of operations.",
+		},
+		IsDataAction: false,
+	},
+	{
+		Name: "Applications.Dapr/register/action",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Dapr",
+			Resource:    "Applications.Dapr",
+			Operation:   "Register Applications.Dapr resource provider",
+			Description: "Registers 'Applications.Dapr' resource provider with a subscription.",
+		},
+		IsDataAction: false,
+	},
+	{
+		Name: "Applications.Dapr/unregister/action",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Dapr",
+			Resource:    "Applications.Dapr",
+			Operation:   "Unregister 'Applications.Dapr' resource provider",
+			Description: "Unregisters 'Applications.Dapr' resource provider with a subscription.",
+		},
+		IsDataAction: false,
+	},
+	{
+		Name: "Applications.Dapr/secretStores/read",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Dapr",
+			Resource:    "secretStores",
+			Operation:   "Get/List daprSecretStores",
+			Description: "Gets/Lists daprSecretStore resource(s).",
+		},
+		IsDataAction: false,
+	},
+	{
+		Name: "Applications.Dapr/secretStores/write",
+		Display: &v1.OperationDisplayProperties{
+			Provider:    "Applications.Dapr",
+			Resource:    "secretStores",
+			Operation:   "Create/Update daprSecretStores",
+			Description: "Creates or updates a daprSecretStore resource.",
```

Comment:
For both Operations and Description, it might be better to say 'Dapr secretStore resource' or 'secretStore resource'. The current text might give the incorrect impression that resource is called daprSecretStore. Thoughts?

Reply:
Update the Description and Opertion properties